app/globals.css

```
@@ -130,6 +130,14 @@
     --color-red-500: #FF495B;
     --color-orange-500: #FF8243;
     --color-purple-500: #D13BFF;
+
+    /* Animations */
+    --animate-slide-down: slideDown 0.2s ease-out;
```

Stylelint (17.12.0)

[error] Expected "slideDown" to be "slidedown" (value-keyword-case)

(value-keyword-case)